driver/drivers_test.go

```diff
@@ -46,6 +54,10 @@ func TestDrivers_Add(t *testing.T) {
 				actual, ok := drivers.drivers[tc.taskName]
 				require.True(t, ok)
 				assert.Equal(t, tc.driver, actual)
+
+				for _, val := range md.templateIDs {
```

Comment:
You might be able to use https://pkg.go.dev/github.com/stretchr/testify/assert#ElementsMatch

Reply:
Oh that's cool! There are definitely a number of tests I could use that for. Thanks for finding that.

Reply:
Although, I'm not sure if it makes sense in this case, I just want to see that map contains a particular value, I can add a comment